opentelemetry-sdk/tests/performance/resource-usage/trace/profile_resource_usage_simple_export.py

```diff
@@ -0,0 +1,47 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from unittest.mock import patch
+
+from opentelemetry.exporter.otlp.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider, sampling
+from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+
+TEST_DURATION_SECONDS = 15
+SPANS_PER_SECOND = 10_000
+
+
+class MockTraceServiceStub(object):
+    def __init__(self, channel):
+        self.Export = lambda *args, **kwargs: None
+
+
+old_stub = OTLPSpanExporter._stub
+OTLPSpanExporter._stub = MockTraceServiceStub
```

Comment:
What does `._stub` do?

Reply:
Nothing special, this is just to mock [the underlying call to the gRPC channel](https://github.com/open-telemetry/opentelemetry-python/blob/cd39fc17b21baf68eddc4ddd0acfe2e80df59eae/exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py#L221-L225) as a **no-op**. We don't want to call a real channel because it would expect a response back from a running Collector (which isn't run during these tests).